Resources/config/doctrine/BasePage.orm.xml

```diff
@@ -22,6 +22,7 @@
         <field name="templateCode" type="string" column="template" nullable="false"/>
         <field name="createdAt" type="datetime" column="created_at"/>
         <field name="updatedAt" type="datetime" column="updated_at"/>
+        <field name="targetRedirectType" type="integer" column="target_redirect_type" nullable="true" default="302"/>
```

Comment:
> probably

@n3o77 please check, I'm really not sure about this. It could make the CI fail though, if like me, you have `doctrine:schema:validate` in CI